pkg/project/api/types.go

```diff
@@ -32,15 +32,11 @@ type Project struct {
 	kapi.TypeMeta
 	kapi.ObjectMeta
 
-	// TODO: remove me
-	DisplayName string
-	Spec        ProjectSpec
-	Status      ProjectStatus
+	Spec   ProjectSpec
+	Status ProjectStatus
 }
 
 type ProjectRequest struct {
 	kapi.TypeMeta
 	kapi.ObjectMeta
-
-	DisplayName string
```

Comment:
I do not understand removing this from a ProjectRequest.

Can you add back or elaborate why you thought it should be removed?

It's not clear to me since if I request a project, the project's display name is part of that request.